### PR TITLE
docs / coinbase connector docs

### PR DESCRIPTION
**Pull request by yzhang1994**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**A description of the changes proposed in the pull request**:
- add in code documentation to coinbase market connector

### CODE REVIEW

File: hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py [hunk 1]
```diff
@@ -132,6 +142,10 @@ def order_book_class(self) -> CoinbaseProOrderBook:
 
     @staticmethod
     async def get_snapshot(client: aiohttp.ClientSession, trading_pair: str) -> Dict[str, any]:
+        """
+        Get a list of active trading pairs from the rest API
```
Comment (chowryan) [inline hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py]: i think this is wrong?

File: hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py [hunk 2]
```diff
@@ -114,9 +115,18 @@ def __init__(self, symbols: Optional[List[str]] = None):
...
+        """
+        *required
+        Get relevant order book class to access class specific methods
+        :returns: OrderBook class
+        """
         return CoinbaseProOrderBook
 
     async def get_trading_pairs(self) -> List[str]:
+        """
+        Get a list of active trading pairs
```
Comment (chowryan) [inline hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py]: maybe mention that if symbols is defined, it just returns that

File: hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py [hunk 3]
```diff
@@ -245,6 +276,12 @@ def order_book_class(self) -> CoinbaseProOrderBook:
                 await asyncio.sleep(30.0)
 
     async def listen_for_order_book_snapshots(self, ev_loop: asyncio.BaseEventLoop, output: asyncio.Queue):
+        """
+        *required
+        Subscribe to snapshot channel via web socket, and keep the connection open for incoming messages
```
Comment (chowryan) [inline hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py]: this function's name might be misleading; seems like it takes a snapshot for each trading pair and puts it in the output queue. Doesn't subscribe to a snapshot channel

File: hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py (hunk 1 above)
Comment (yzhang1994) [reply hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py]: fixed.

File: hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py (hunk 2 above)
Comment (yzhang1994) [reply hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py]: fixed.

File: hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py (hunk 3 above)
Comment (yzhang1994) [reply hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py]: fixed.

**Pull request merged**